vmdb/app/models/miq_queue.rb

```diff
@@ -1,6 +1,28 @@
 require 'timeout'
 require 'digest'
 
+# Message Bus entry to run a method on any server
+# segmentation:
+#   region
+#      this is partitioned by database, so no need to reference)
+#   zone
+#     this is a portion of the database, most messages are performed in the region of the requester
+#     put: default to myzone. pass in nil to be performed in any zone
+#     get: uses [nil, zone] to fetch my zone and an or for any zone
+#   role
+#     typically nil
+#     put: default to nil
+#     get: use [nil, role] to fetch for my role or any role
+#   queue_name
+#     put: default to "generic"
+#     get: use nil,
+#
+
+# TODO: give justification for all fields
+# TODO: pull priority class out int miq_priority?
+# TODO: add priority scope, and task_id - use AR for everything else
+# TODO: test putting in passing in a zone
+
```

Comment:
yes. Thanks.

Think I'll just pull these comments out. All in all it doesn't help and makes things a little confusing.